packages/app/src/pages/SwapPage/PricePerToken.tsx

```diff
@@ -20,7 +20,7 @@ function getPricePerToken(
   toAmount?: bigint | null
 ) {
   if (!toAmount || !fromAmount) return "";
-  const ratio = divideBigInt(toAmount, fromAmount);
+  const ratio = divideFnValidOnly(toAmount, fromAmount);
```

Comment:
What is valid only?

Reply:
- `divideFn` do the raw division with the params you informed, which may result in invalid values, like `NaN` `Infinity` or even `undefined` 

- `divideFnValidOnly` handles inputs first, then do the raw division, then handle the output. Making sure that result is always gonna be a valid `number`